definitions/json/Categories/Categories-CaseFileView-nonprod.json

```diff
@@ -0,0 +1,485 @@
+[
```

Comment:
I think you are referring to these changes in CaseField and ComplexTypes. If so, you need to put them in a prod file too.

Reply:
Done